packages/appeals-service-api/src/routes/v2/events/repo.js

```diff
@@ -0,0 +1,61 @@
+const { createPrismaClient } = require('#db-client');
+
+/**
+ * @typedef {import('pins-data-model/src/schemas').AppealEvent} DataModelEvent
+ * @typedef {import('@prisma/client').Event} PrismaEvent
+ */
+
+/**
+ * @param {DataModelEvent} dataModelEvent
+ * @returns {Omit<PrismaEvent, 'internalId'>}
+ */
+const mapDataModelToFODBEvent = (dataModelEvent) => ({
+	id: dataModelEvent.eventId,
+	type: dataModelEvent.eventType,
```

Comment:
I modelled the DB like this:
	/** type of event */
	type: 'siteVisit' | 'hearing' | 'inquiry' | 'inHouse' | 'preInquiry';
	/** additional event type detail */
	subtype: 'accompanied' | 'unaccompanied' | 'accessRequired' | 'virtual';
	
	where as the type comes through like this:
	"enum": [
        "site_visit_access_required",
        "site_visit_accompanied",
        "site_visit_unaccompanied",
        "hearing",
        "hearing_virtual",
        "inquiry",
        "inquiry_virtual",
        "in_house",
        "pre_inquiry",
        "pre_inquiry_virtual"
      ]
      
      Need to either update the logic to work without the subtype or map here

Reply:
ah I see, sorry didn't appreciate this, I'll write some code to chop it up